files/es/web/javascript/reference/global_objects/weakmap/index.md

```diff
@@ -0,0 +1,147 @@
+---
```

Comment:
@davbrito , @Vallejoanderson , @JuanVqz, @cw118

Does somebody know why this file was added to my pr? I don't recall to have made this action.

Alguien me podría ayudar a entender el por qué se agregó éste archivo a mi PR? no recuerdo haber hecho esta acción.

Reply:
I'm guessing it has something to do with trying to merge into the wrong branch at the beginning. This file was added since the changes JuanVqz made to it on his branch (`update-weakmap-es`) carried over when you switched the base branch to `main`.

Reply:
Yep, I thought the same, I guess if we merge my branch first it will disappear.
Otherwise, it will cause conflicts on one of our branches.

Reply:
I understand, I will delete these files from my PR. I think that will avoid the conflict risk.